test/HttpTestHelpers.h

```diff
@@ -39,21 +39,21 @@ class TestHttpServer {
   // Indicator whether the server has been shut down. We need this because
   // `HttpServer::shutDown` must only be called once.
   std::atomic<bool> hasBeenShutDown_ = false;
-  ad_utility::websocket::QueryRegistry registry_;
 
  public:
   // Create server on localhost. Port 0 instructs the operating system to choose
   // a free port of its choice.
   explicit TestHttpServer(HttpHandler httpHandler) {
     const std::string& ipAddress = "0.0.0.0";
     int numServerThreads = 1;
-    auto webSocketSessionSupplier = [this](net::io_context& ioContext) {
+    auto webSocketSessionSupplier = [](net::io_context& ioContext) {
       ad_utility::websocket::QueryHub queryHub{ioContext};
-      return [this, queryHub = std::move(queryHub)](
+      ad_utility::websocket::QueryRegistry registry;
+      return [queryHub = std::move(queryHub), registry = std::move(registry)](
```

Comment:
```suggestion
      using namespace ad_utility::websocket;
       return [queryHub = QueryHub{ioContext}, registry = QueryRegistry{}](
```
(Also replaces line 50). No need to `move` if you can directly construct in place.